### PR TITLE
Add transparent vue svg

### PR DESCRIPTION
**Pull request by crobibero**
Just opened in inkscape and deleted the background

### CODE REVIEW

**Review by ThibaultNocchi — APPROVED** [review]
LGTM

**Review by MrTimscampi — CHANGES_REQUESTED** [review]
Might want to save as Plain SVG instead of Inkscape SVG.

**Review by MrTimscampi — APPROVED** [review]

File: branding/SVG/icon-transparent-vue.svg [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <path
+       id="inner-shape"
+       d="M256,201.62c-20.44,0-86.23,119.29-76.2,139.43s142.48,19.92,152.4,0S276.47,201.63,256,201.62Z"
+       fill="url(#linear-gradient)" />
+    <path
+       id="outer-shape"
+       d="M256,23.3C194.44,23.3-3.82,382.73,26.41,443.43s429.34,60,459.24,0S317.62,23.3,256,23.3ZM406.51,390.76c-19.59,39.33-281.08,39.77-300.89,0S215.71,115.48,256.06,115.48,426.1,351.42,406.51,390.76Z"
+       fill="url(#linear-gradient)" />
+  </g>
+</svg>
```
Comment (nielsvanvelzen) [inline branding/SVG/icon-transparent-vue.svg:55]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><defs><linearGradient id="a" x1="110.25" y1="213.3" x2="496.14" y2="436.09" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#41b883"/><stop offset="1" stop-color="#34495e"/></linearGradient></defs><g fill="url(#a)"><path d="M256 201.62c-20.44 0-86.23 119.29-76.2 139.43s142.48 19.92 152.4 0-55.73-139.42-76.2-139.43z"/><path d="M256 23.3c-61.56 0-259.82 359.43-229.59 420.13s429.34 60 459.24 0S317.62 23.3 256 23.3zm150.51 367.46c-19.59 39.33-281.08 39.77-300.89 0s110.09-275.28 150.44-275.28S426.1 351.42 406.51 390.76z"/></g></svg>
```

Optimized it a bit with `svgo`...

Comment (nielsvanvelzen) [reply branding/SVG/icon-transparent-vue.svg:55]: I don't think we did this with most assets in this repo so you may ignore this :p

**Pull request merged**